controllers/articles.controllers.js

```diff
@@ -20,3 +21,13 @@ exports.getArticleById = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.patchArticleById = (req, res, next) => {
+  const { article_id } = req.params;
+  const { votes } = req.body;
+  return patchArticle(votes, article_id)
+    .then((updatedArticle) => {
+      res.status(202).send({ updatedArticle });
```

Comment:
I'd go with a 200 here since the request is okay and 202 is more for if a request is processing but hasn't been completed.